src/main/java/Deadlines.java

```diff
@@ -0,0 +1,24 @@
+public class Deadlines extends Task {
+    private String deadline;
+    public Deadlines(String description, String deadline) {
+        super(description);
+        this.deadline = deadline;
+    }
+
+    public Deadlines(String input) {
+        super(input.substring(input.indexOf(" ") + 1, input.indexOf("/by") - 1));
+        this.deadline = input.substring(input.indexOf("/by") + 4);
```

Comment:
In the spirit of SLAP, perhaps a function could be created to obtain the deadline?